frontend/js/src/explore/huesound/HueSound.tsx

```diff
@@ -0,0 +1,205 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+
+import * as React from "react";
+import { get, has } from "lodash";
+import * as tinycolor from "tinycolor2";
+import { toast } from "react-toastify";
+import { Helmet } from "react-helmet";
+import { Link } from "react-router-dom";
+import ColorWheel from "./components/ColorWheel";
+import { convertColorReleaseToListen } from "./utils/utils";
+import GlobalAppContext from "../../utils/GlobalAppContext";
+
+import BrainzPlayer from "../../common/brainzplayer/BrainzPlayer";
+import Loader from "../../components/Loader";
+import ListenCard from "../../common/listens/ListenCard";
+import Card from "../../components/Card";
+import { COLOR_WHITE } from "../../utils/constants";
+import { ToastMsg } from "../../notifications/Notifications";
+
+export default function HueSound() {
+  const { APIService } = React.useContext(GlobalAppContext);
+  const [loading, setLoading] = React.useState(false);
+  const [colorReleases, setColorReleases] = React.useState<ColorReleaseItem[]>(
+    []
+  );
+  const [selectedRelease, setSelectedRelease] = React.useState<
+    ColorReleaseItem
+  >();
+  const [selectedColorString, setSelectedColorString] = React.useState<
+    string
+  >();
+  const [gridBackground, setGridBackground] = React.useState<string>(
+    COLOR_WHITE
+  );
+
+  React.useEffect(() => {
+    setLoading(true);
+    const hex = tinycolor(selectedColorString).toHex(); // returns hex value without leading '#'
+    APIService.lookupReleaseFromColor(hex)
+      .then((newColorReleases) => {
+        const { releases } = newColorReleases.payload;
+        const lighterColor = tinycolor(selectedColorString).lighten(40);
+        setColorReleases(releases);
+        setGridBackground(lighterColor.toRgbString());
+      })
+      .catch((err) => {
+        toast.error(
+          <ToastMsg
+            title="Error"
+            message={err.message ? err.message.toString() : err.toString()}
+          />,
+          { toastId: "error" }
+        );
+      });
+  }, [selectedColorString, APIService]);
```

Comment:
Set the loading state to false once the API call have been made
```suggestion
    setLoading(false);
```